src/components/AssetInfo/TokenSideBarContent.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import css from "./TokenSideBar.scss";
+import TokenSideBarHolder from "./TokenSideBarHolder";
+import TokenSideBarBeneficiary from "./TokenSideBarBeneficiary";
+import TokenSideBarNoMatch from "./TokenSideBarNoMatch";
+import { transferTokenOwnership, transactionMinedReceipt } from "../../services/token";
+
+interface TokenSideBarContentProps {
+  adminAddress: string;
+  beneficiaryAddress: string;
+  holderAddress: string;
+  approvedBeneficiaryAddress: string;
+  registryAddress?: string;
+}
+
+const TokenSideBarContent = ({
+  adminAddress,
+  beneficiaryAddress,
+  holderAddress,
+  approvedBeneficiaryAddress,
+  registryAddress
+}: TokenSideBarContentProps) => {
+  const [fieldValue, setFieldValue] = useState({ newHolder: "", approvedBeneficiary: "" });
```

Comment:
The `TokenSideBarHolder` should handle it's own state of the inputs. Think about how to put ` TokenSideBarHolder` in a storybook, it will not be able to handle input changes. 

I will imagine that we pass in functions such as `handleTransferOwnership(newAddress)` to the `TokenSideBarHolder`.